mflowgen/glb_tile/construct.py

```diff
@@ -107,6 +112,8 @@ def construct():
   sram_steps = \
     [synth, iflow, init, power, place, cts, postcts_hold, \
      route, postroute, postroute_hold, signoff, genlib]
+  if multicorner:
+      sram_steps.append(genlib_ff)
   for step in sram_steps:
     step.extend_inputs( ['sram_tt.lib', 'sram.lef'] )
```

Comment:
Something here doesn't look right for genlib_ff. It is getting an sram_tt.lib input now.

Reply:
It's already getting all corners for the other cells in the design... I think this should be ok, but I can change it.

Reply:
I mean it's not getting sram_ff.lib? And genlib_ff does not need lef right?

Reply:
oh right.. missed that. Adding now